src/tests/3-corax+.8o

```diff
@@ -247,14 +276,38 @@
   #test Fx33
   y += 5
   drawop imF im3
+  
+  # >= 100
   i := scratchpad
   v6 := 137
   bcd v6
   load v2
   i := image-ok
-  if v0 != 1 then i := image-no
-  if v1 != 3 then i := image-no
-  if v2 != 7 then i := image-no
+  if v0 != 1 begin i := image-no jump skipOtherFx33Tests
+  if v1 != 3 begin i := image-no jump skipOtherFx33Tests
+  if v2 != 7 begin i := image-no jump skipOtherFx33Tests
```

Comment:
You can't have a `begin` without an `end`. Also, I usually stay away from putting multiple instructions on the same line, for readability. So this either has to be:

```python
  if v0 != 1 begin
    i := image-no
    jump skipOtherFx33Tests
  end
  if v1 != 3 begin
    i := image-no
    jump skipOtherFx33Tests
  end
  if v2 != 7 begin
    i := image-no
    jump skipOtherFx33Tests
  end
```

Or, what's probably a bit nicer, and certainly smaller:

```python
  i := image-no # This once, at the top

  # ... other code

  if v0 != 1 then jump skipOtherFx33Tests
  if v1 != 3 then jump skipOtherFx33Tests
  if v2 != 7 then jump skipOtherFx33Tests

  # ... other tests

  i := image-ok # this once, at the bottom, just before the label
```